### ISSUE TITLE
[BUG] Clicking volume icon does nothing

### ISSUE THREAD

**Issue by softkittendreamforever** [dremin/RetroBar#939]
After updating to v1.16.34, clicking the volume icon in the tray doesn't bring up the volume slider. Right-click also does not work. All other icons work as expected.
I have tried to get it working again by closing OpenShell, restarting RetroBar, updating and restarting my computer...but it is persistent, it happens every time I start RetroBar.

(I am using Windows 10, if that is relevant.)

**Comment by alt44s** [comment]
I have the same issue, however it only happens after RetroBar has been running for a while. The volume button works fine at app start, but after some time it seems to stop working. Restarting does help, but it's a bit inconvenient.

**Comment by velissarious** [comment]
I have the same issue with the 32-bit edition, version 1.17.71+916bdca1ef .
Please fix this issue.

**Comment by velissarious** [comment]
I think that the issue has to do with the specific type of version of the .NET 6.0 Runtime for RetroBar you need the:

.NET 6.0 **Desktop** Runtime (v6.0.36) - Windows x64 Installer

I already had a different version for console apps.

Now the sound volume icon works fine!

**Comment by xoascf** [comment]
> I think that the issue has to do with the specific type of version of the .NET 6.0 Runtime for RetroBar you need the:
> 
> .NET 6.0 **Desktop** Runtime (v6.0.36) - Windows x64 Installer
> 
> I already had a different version for console apps.
> 
> Now the sound volume icon works fine!

No, it's not that, I had it installed and still clicking (or even right-clicking) on the volume icon does nothing, all other icons work though:
![DesktopRuntimeAlreadyInstalled](https://github.com/user-attachments/assets/7739bf95-3a50-482a-b9b4-f4b9379ef161)

I had to restart `explorer.exe` using Task Manager to get it to work again.
